app/js/components/breadCrumbs/breadCrumb.js

```diff
@@ -7,9 +7,27 @@ import { Link, browserHistory, withRouter } from 'react-router';
 class BreadCrumbComponent extends React.Component {
   constructor(props) {
     super(props);
+    this.handleRoute=this.handleRoute.bind(this)
+  }
+  handleRoute(e){
+    if(e.currentTarget.dataset.id == "search"){
+      var current_url=this.props.location.pathname;
```

Comment:
nitpick: You would probably want to stick to strictly ES6 syntax by using `let` and `const`. They are less likely to have side effects in your code.